testnet/stacks-node/src/neon_node.rs

```diff
@@ -650,18 +650,22 @@ impl InitializedNeonNode {
             .expect("Unexpected BurnDB error fetching block commits");
         for op in block_commits.into_iter() {
             if op.txid == block_snapshot.winning_block_txid {
+                info!("Received burnchain block #{} including block_commit_op (winning) - {}", block_height, op.input.to_address());
                 last_sortitioned_block = Some((block_snapshot.clone(), op.vtxindex));
                 // Release current registered key if leader won the sortition
                 // This will trigger a new registration
                 if op.input == self.burnchain_signer {
                     won_sortition = true;
                 }    
-            }            
+            } else {
+                info!("Received burnchain block #{} including block_commit_op - {}", block_height, op.input.to_address());
+            }
         }
 
         let key_registers = BurnDB::get_leader_keys_by_block(&ic, block_height, burn_hash)
             .expect("Unexpected BurnDB error fetching key registers");
         for op in key_registers.into_iter() {
+            info!("Received burnchain block #{} including key_register_op - {}", block_height, op.address);
```

Comment:
Same here -- can this be a debug log?

Reply:
While I agree that logging each and every burn operation can be verbose, I think it's helpful to at least log the following per burnchain block:

* That the burnchain block was processed
* What the winning Stacks block hash was (if there was a sortition)

This has the effect of showing the user that the node is making progress, even though it hasn't begun to download any Stacks blocks yet (synchronizing with the Bitcoin chain can take a while -- like 15 minutes or so -- so some kind of "I'm alive!" hint on the console would be desired IMHO).

Reply:
Any chance we can revisit the level of these logs when we have many miners and that it is a problem? We have one miner today. These logs are needed in Kibana, and we can't, **today**, start a master node with `BLOCKSTACK_DEBUG=1` (@wileyj can chime in for details).

Reply:
I think that this would log on a follower as well.

Reply:
Yup, it would

Reply:
You don't want to have these logs if you're running a follower?

Reply:
It just seems like a lot of logs! This would add like 2 new log lines every 7 seconds. But it's okay with me if we include these for now.

Reply:
We updated the block time last week - now 30s.
I just added a filter - only showing this logs if you're a miner.